### DISCUSSION TITLE
Allow overriding styles when using styled on a styled component

### DISCUSSION THREAD

**Discussion by broccolai** [chakra-ui/panda#874] [Ideas]
### Description

Currently, with panda when you declare a style with in child component, it will not always use that value over the value provided by the parent. 

[Example here](https://github.com/broccolai/panda-inheritence-quirk/blob/master/src/App.tsx#L6)

### Problem Statement/Justification

This behavior makes sense and is found in similar libraries such as [styled components](https://styled-components.com/docs/basics#styling-any-component) 

### Proposed Solution or API

I have looked at how other libraries implement this but their methods cannot be applied to Panda because of how it's structured.

**Comment by astahmer** [comment]
it's been a while but I just found this issue again so I'm answering now that we have a solution for it https://panda-css.com/docs/concepts/writing-styles#merging-styles if others need it
